src/components/Bg/BgStyles.jsx

```diff
@@ -7,7 +7,7 @@ export const BgStyles = styled.section`
   position: relative;
   margin-bottom: 60px;
   margin-top: 100px;
-  background: url(${BgImg});
+  background: #F4F4F4;
```

Comment:
we should have a colour variable defined for this

Reply:
> we should have a colour variable defined for this

I can create a variable. I thought it wasn't worth it since we're only gonna use this colour once. what do u guys think?

Reply:
I think we should have the entire grayscale as defined color variables if not there already

Reply:
Yeah, colors should be same from dashboard to static site, no hardcoded colors

Reply:
will fix